api.visual/src/org/netbeans/api/visual/widget/Scene.java

```diff
@@ -592,6 +593,26 @@ public final Rectangle convertSceneToView (Rectangle sceneRectangle) {
                 (double) sceneRectangle.height * zoomFactor));
     }
 
+    /**
+     * Converts a rectangle in the view coordination system into the scene one. It is just the opposite to 
+     * {@link #convertSceneToView(java.awt.Rectangle)}.
+     * @param viewRect the rectangle, in view coordinates
+     * @return the same rectangle, in scene coordinates
+     * @since 2.48
```

Comment:
This one is using `@since 2.48` the other methods I have seen so far use `2.49`. Is that intended?

Reply:
Typo; will fix, thanks